crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip.rs

```diff
@@ -277,14 +278,20 @@ pub struct ShardedGossipLocalState {
     local_agents: HashSet<Arc<KitsuneAgent>>,
     tuning_params: KitsuneP2pTuningParams,
     /// If Some, we are in the process of trying to initiate gossip with this target.
-    initiate_tgt: Option<(GossipTgt, u8)>,
+    initiate_tgt: Option<(GossipTgt, u32)>,
     round_map: RoundStateMap,
     /// Metrics for a connection.
     // FIXME: Currently the p2p metric store is setup to track
     // metrics per agent but we need to track metrics per connection.
     metrics: HashMap<Tx2Cert, MetricInfo>,
     /// Last moment we locally synced.
     last_local_sync: Option<std::time::Instant>,
+    /// Trigger local sync to run on the next iteration.
+    trigger_local_sync: bool,
```

Comment:
This value doesn't seem to be read anywhere but maybe that's coming in future work?

Reply:
It's read in the `should_local_sync` function